#! /bin/sh /usr/share/dpatch/dpatch-run
## 70_zc-locale_472300-fix.dpatch by Sebastien Delafond <seb@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Rename locale to zc-locale to avoid conflict with liblocale-ruby.

@DPATCH@
Index: zonecheck-3.0.0/zc/zc.rb
===================================================================
--- zonecheck-3.0.0.orig/zc/zc.rb	2010-06-24 11:19:54.000000000 +0200
+++ zonecheck-3.0.0/zc/zc.rb	2010-06-24 11:21:16.000000000 +0200
@@ -192,7 +192,7 @@
 
 
 # ZoneCheck component
-require 'locale'
+require 'zc-locale'
 require 'msgcat'
 require 'console'
 require 'zonecheck'
